plugins/modules/ec2_launch_template.py

```diff
@@ -367,16 +409,37 @@
         version_added: 3.1.0
         type: str
         description: >
-          - Wether the instance metadata endpoint is available via IPv6 (C(enabled)) or not (C(disabled)).
+          - Wether the instance metadata endpoint is available via IPv6 (V(enabled)) or not (V(disabled)).
         choices: [enabled, disabled]
         default: 'disabled'
       instance_metadata_tags:
         version_added: 3.1.0
         type: str
         description:
-          - Wether the instance tags are availble (C(enabled)) via metadata endpoint or not (C(disabled)).
+          - Wether the instance tags are availble (V(enabled)) via metadata endpoint or not (V(disabled)).
         choices: [enabled, disabled]
         default: 'disabled'
+  template_tags:
```

Comment:
I guess we should go with `tags` name instead (this ensures consistency). I guess for this option and the purge_tags we can extend the doc fragments.

@tremble @markuman Any thoughts?

Reply:
we can do that but this will be a breaking change.

Reply:
Why would it be a breaking change?

Reply:
But the launch template data tags are `tag_specifications.tags`, right?  So, to tag a launch template resource you will need to only use `tags` (the same way you use to tag the other resources) and `purge_tags`. They do not have anything to do with `tag_specifications.tags`, that's why i was suggesting to rename and be consistent with the way we use tags option in the other modules.

Reply:
updated